core/code/ornaments.js

```diff
@@ -27,49 +29,154 @@ window.ornaments = {
       layerGroup = L.canvasIconLayer;
       L.CanvasIconLayer.mergeOptions({ padding: L.Canvas.prototype.options.padding });
     }
+    this.load();
     this._layer = layerGroup();
     this._beacons = layerGroup();
     this._frackers = layerGroup();
+    this._scout = layerGroup();
+    this._battle = layerGroup();
+    this._excluded = layerGroup(); //to keep excluded ornaments in an own layer
+    
     window.layerChooser.addOverlay(this._layer, 'Ornaments');
     window.layerChooser.addOverlay(this._beacons, 'Beacons');
     window.layerChooser.addOverlay(this._frackers, 'Frackers');
+    window.layerChooser.addOverlay(this._scout, 'Scouting');
+    window.layerChooser.addOverlay(this._battle, 'Battle');
+    window.layerChooser.addOverlay(this._excluded, 'Excluded'); //just for testing
+
+    $('#toolbox').append('<a onclick="window.ornaments.ornamentsOpt();return false;" accesskey="o" title="Edit ornament exclusions [o]">Ornaments Opt</a>');
+
   },
 
   addPortal: function (portal) {
     this.removePortal(portal);
-
     var ornaments = portal.options.data.ornaments;
     if (ornaments && ornaments.length) {
       this._portals[portal.options.guid] = ornaments.map(function (ornament) {
-        var layer = this._layer;
+        var layer = this._layer; 
+        var opacity = this.OVERLAY_OPACITY;
+        var size = this.OVERLAY_SIZE;
+        var anchor = [size / 2, size / 2];
+        var iconUrl = '//commondatastorage.googleapis.com/ingress.com/img/map_icons/marker_images/' + ornament + '.png';
+
         if (ornament.startsWith('pe')) {
           layer = ornament === 'peFRACK'
             ? this._frackers
             : this._beacons;
         }
-        var size = this.OVERLAY_SIZE;
+
+        if (ornament.startsWith('sc')) {
+          layer = this._scout;
+        }
+
+        if (ornament.startsWith('peB')) {
+          layer = this._battle;
+        }
+
+        if (typeof (window.ornaments.iconUrls[ornament]) !== 'undefined') {
+          opacity = 1;
```

Comment:
Things might evolve in future, but is there really a need for generic markers? Everyone is looking into fancy markers, but the real issue is the load of markers that are shown during an anomaly.

So initial issue was to hide/exclude ornaments from showing up on intel. Kythera Phase 3 was pretty messy with a lot of ornaments still showing from last rounds, so I looked for an easy way to get rid of them.

But as the existing plugins added iconUrls, this seemed to be "wanted". The settings for offset or opacity were simply taken from there.

What value for opacity would you prefer?

But what would a generic marker offer to the user?
I do not want to implement a lot of stuff, because "it can be done".